plugins/kubernetes-backend/src/kubernetes-auth-translator/KubernetesAuthTranslatorGenerator.test.ts

```diff
@@ -14,46 +14,78 @@
  * limitations under the License.
  */
 
-import { KubernetesAuthTranslator } from './types';
-import { GoogleKubernetesAuthTranslator } from './GoogleKubernetesAuthTranslator';
 import { KubernetesAuthTranslatorGenerator } from './KubernetesAuthTranslatorGenerator';
-import { NoopKubernetesAuthTranslator } from './NoopKubernetesAuthTranslator';
-import { AwsIamKubernetesAuthTranslator } from './AwsIamKubernetesAuthTranslator';
-import { OidcKubernetesAuthTranslator } from './OidcKubernetesAuthTranslator';
-import { getVoidLogger } from '@backstage/backend-common';
+import { ClusterDetails } from '../types';
+import { KubernetesRequestAuth } from '@backstage/plugin-kubernetes-common';
 
-const logger = getVoidLogger();
+const mockTranslator = jest.fn();
```

Comment:
Sure thing will remove the module scoped mock declaration into the describe block

Reply:
Was there a preference for which name? I sort of like DispatchingKubernetesAuthTranslator since the plural KubernetesAuthTranslators implies a collection of translators which this class isnt.

Reply:
Done